src/api/public/apidocs/paths/label_templates.yaml

```diff
@@ -0,0 +1,53 @@
+get:
+  summary: List global label templates.
+  description: |
+    **(Unstable)** List the label templates globaly available.
+
+    This is only for admins.
+  security:
+    - basic_authentication: []
```

Comment:
When testing this via the `/apidocs` interface in my local instance, I could get a list of the existing global labels without being logged in, but when doing the same via console using the provided curl instruction, it denies the access... Something's fishy...

Reply:
You shouldn't get the list of global templates from `/apidocs` without being logged in, in first instance. This should be cookies. Please try again going to http://localhost:3000 and logging out.

Via console and curl, did you try with an admin user?

Reply:
> You shouldn't get the list of global templates from `/apidocs` without being logged in, in first instance. This should be cookies. Please try again going to http://localhost:3000 and then log out.

Cookies... that was the issue... In an incognito window works perfectly.

> Via console and curl, did you try with an admin user?

Regular user is banned, Admin works fine